lib/presentation/widget/shared_space_details/add_shared_space_member/add_shared_space_member_widget.dart

```diff
@@ -206,22 +212,48 @@ class _AddSharedSpaceMemberWidgetState extends State<AddSharedSpaceMemberWidget>
                     member.account.name, member.account.mail, member.role.name.getRoleName(context),
                     tileColor: Colors.white,
                     onSelectedRoleCallback: () =>
-                        selectRoleBottomSheet(context, member.role.name, (newRole) {
-                          _model.changeMemberRole(sharedSpace.sharedSpaceId, member, newRole);
-                        })).build();
+                        selectRoleBottomSheet(
+                          context, 
+                          member.role.name,
+                          onNewRoleUpdated: (newRole) {
+                            _model.changeMemberRole(sharedSpace.sharedSpaceId, member, newRole);
+                          }),
+                    onDeleteMemberCallback: () => confirmDeleteMember(
```

Comment:
one more remark, only `Admin` can delete a member.

![image](https://user-images.githubusercontent.com/6462404/115312465-9367fa80-a19b-11eb-98a3-cb493e86c8a9.png)

Are you sure `delete` button only visible in the case of `Admin`? (same for edit role)

please check the details on: https://connectors.api.twake.app/only_office/spreadsheet/editor?workspace_id=f339d54a-e833-11ea-92c3-0242ac120004&group_id=56393af2-e5fe-11e9-b894-0242ac120004&file_id=21c8447a-1825-11eb-8ce0-0242ac120004&preview=&user_token=0%2F0S5qeLYvUyO1MIEx595ltb5H4yr6L4iLi8EXQz%2BJwa23Gs

Reply:
It's my fault. Sorry about this. Will fix ASAP